modules/beta-private-cluster/variables.tf

```diff
@@ -393,6 +393,14 @@ variable "master_ipv4_cidr_block" {
   default     = "10.0.0.0/28"
 }
 
+variable "master_global_access_enabled" {
+  type        = bool
+  description = "(Beta) Whether the cluster master is accessible globally (from any region) or only within the same region as the private endpoint."
+
+  default = true
```

Comment:
I dont have strong feelings but since provider default if false, I wonder if we should keep it that way. WDYT @ilkelma @morgante

Reply:
Yeah that's my bad, we should definitely match the provider default imho. Will push a fix.

Reply:
Actually I like defaulting to true. I think it's the best practice.

Reply:
yeah there's kinda bad feels in either direction "why can't I connect to my cluster?" vs "why did something get set that I wasn't expecting". But given that modules are generally collections of best practices I can see the logic in avoiding the former problem as the latter will likely be much rarer. I'll push a fix flipping it back.